etc/picongpu/juwels-jsc/booster_picongpu.profile.example

```diff
@@ -0,0 +1,126 @@
+export PIC_PROFILE=$(cd $(dirname $BASH_SOURCE) && pwd)"/"$(basename $BASH_SOURCE)
+
+# User Information ################################# (edit the following lines)
+#   - automatically add your name and contact to output file meta data
+#   - send me a mail on batch system jobs: NONE, BEGIN, END, FAIL, REQUEUE, ALL,
+#     TIME_LIMIT, TIME_LIMIT_90, TIME_LIMIT_80 and/or TIME_LIMIT_50
+export MY_MAILNOTIFY="NONE"
+export MY_MAIL="someone@example.com"
+export MY_NAME="$(whoami) <$MY_MAIL>"
+
+# Project Information ######################################## (edit this line)
+#   - project account for computing time
+#   - using NF - number of fields, use the last group available
+#export proj=$(groups | awk '{print $(NF)}')
+export proj=ea_picongpu
+# Text Editor for Tools ###################################### (edit this line)
+#   - examples: "nano", "vim", "emacs -nw", "vi" or without terminal: "gedit"
+#export EDITOR="vim"
+
+# Set up environment, including $SCRATCH and $PROJECT
+jutil env activate -p $proj
+
+# General modules #############################################################
+#
+# Requires module stages to be loaded
+OTHERSTAGES=/p/software/juwelsbooster/otherstages
+module purge
+
+module use $OTHERSTAGES
+module load Stages/2020
+
+module load GCC/9.3.0
+module load CMake/3.18.0
+module load ParaStationMPI/5.4.7-1
+module load mpi-settings/default
+#module load mpi-settings/CUDA 
+module load Python/3.8.5
+
+module load Boost/1.74.0
+module load HDF5/1.10.6
+echo "Loaded modules"
+# Other Software ##############################################################
+#
+
+# Define the path to manually installed libraries
+PARTITION_LIB=$PROJECT_boosterea/lib
+
+LIBSPLASH_VER=1.7.0
+PNGWRITER_VER=0.7.0
+BLOSC_VER=1.19.0
+ADIOS_VER=1.13.1
+
+LIBSPLASH_ROOT=$PARTITION_LIB/LIBSPLASH/$LIBSPLASH_VER
+PNGWRITER_ROOT=$PARTITION_LIB/PNGWRITER/$PNGWRITER_VER
+export CMAKE_PREFIX_PATH=$LIBSPLASH_ROOT:$PNGWRITER_ROOT:$CMAKE_PREFIX_PATH
+
+BLOSC_ROOT=$PARTITION_LIB/BLOSC/$BLOSC_VER
+export CMAKE_PREFIX_PATH=$BLOSC_ROOT:$CMAKE_PREFIX_PATH
+export LD_LIBRARY_PATH=$BLOSC_ROOT/lib:$LD_LIBRARY_PATH
+
+ADIOS_ROOT=$PARTITION_LIB/ADIOS/$ADIOS_VER
+export PATH=$ADIOS_ROOT/bin:$PATH
+export CMAKE_PREFIX_PATH=$ADIOS_ROOT:$CMAKE_PREFIX_PATH
+
+
+# Environment #################################################################
+#
+
+export PICSRC=$HOME/shared/src/picongpu-dev-04092020
+export PIC_EXAMPLES=$PICSRC/share/picongpu/examples
+export PIC_BACKEND="cuda:80" # Nvidia A100 architecture
+
+export PATH=$PATH:$PICSRC
+export PATH=$PATH:$PICSRC/bin
+export PATH=$PATH:$PICSRC/src/tools/bin
+
+export PYTHONPATH=$PICSRC/lib/python:$PYTHONPATH
+
+# "tbg" default options #######################################################
+#   - SLURM (sbatch)
+#   - "gpus" queue
+export TBG_SUBMIT="sbatch"
+export TBG_TPLFILE=$HOME"shared/jobScriptTemplates/juwels_booster.tpl"
+#export TBG_TPLFILE="etc/picongpu/juwels-jsc/booster.tpl"
```

Comment:
The profile you uploaded is is the later path, but the one you are using is in the first path. Are they identical? If yes, you could switch back to the default with the next update (not urgent)